docs/user/administration/users/group-types.md

```diff
@@ -0,0 +1,99 @@
+
+![Create Group Types Menu](../../assets/group-type-menu.png)
+
+### Add Group Type:
+
+You'll see in the screenshot below, the form to add your group type. Here are what inputs are used for within BuddyPress:
+
+#### Group Type ID
+
+It's your unique identifier for your group type. It's a required field and you need to use regular & lower-case characters to fill this input. As this value is used as a fallback to the URL portion of the Group type directory link, make sure to choose wisely this identifier.
+
+#### Singular name
+
+This name will be shown in various places such as the group's metabox to set their group type (which is available in the WP-Admin extended profile page) or the group's header of their front-end profile pages (if the Show on Group checkbox is activated).
+
+#### Plural name
+
+It's the plural form of the group type name.
+
+#### Has Directory View
+
+When this checkbox is active, you'll be able to filter the Groups directory according to a specific group type. FYI, a Group Type URL looks like this `https://site.url/groups/type/groupTypeID`. If you need to use a different URL portion to list groups having a specific Group Type inside the Groups directory, you can use the Custom type directory slug input that appears as soon as you activate the Has Directory View checkbox.
+
+#### Show on Group Creation
+
+Activate this checkbox if you want to show this group type during group creation and when a group admin is on the group’s settings page.
+
+#### Show on Group
+
+Activate this checkbox if you want the Group's Group Type information to be displayed on their front-end profile pages header.
+
+![Add Group Type](../../assets/add-group-type.png)
+
+Once saved, an admin notice at the top of the screen will confirm you successfully added a new group type 👏. The table on the right part of the screen will contain a new specific row so that you can edit or remove this group type.
+
+![Group Type Added](../../assets/add-group-type-success.png)
+
+### Edit Group Type:
+
+![Edit Group Type](../../assets/edit-group-type.png)
+
+When you need to edit a group type, go back to the Group types administration of your dashboard to find the table listing all available group types. Hover the group type ID you need to edit so that action links will appear right under this ID. Click on the Edit link to reach the Edit form shown below.
+
+![Update Group Type](../../assets/update-form-group-types.png)
+
+Once you're satisfied with your changes, click on the "Update" blue button to save them. The page will be reloaded to display the same form with applied changes and an Admin notice at the top of the screen will confirm you successfully updated your group type properties. The link inside this notice brings you back to the screen listing all available group types.
+
+![Group Type Updated](../../assets/edit-group-type-success.png)
+
+### Delete Group Type:
+
```

Comment:
```suggestion

Deleting a group type is very similar to editing a group type. The only difference will be that instead of clicking on the "Edit" link, you'll click on the "Delete" one. 

```